cmd/icinga-kubernetes/main.go

```diff
@@ -97,8 +97,11 @@ func main() {
 	log := klog.NewKlogr()
 
 	var cfg daemon.Config
-	err = config.FromYAMLFile(configLocation, &cfg)
-	if err != nil {
+
+	if err = config.Load(&cfg, config.LoadOptions{
+		Flags:      glue,
+		EnvOptions: config.EnvOptions{Prefix: "IFK_"},
```

Comment:
Although there is (much) more to type, `ICINGA_FOR_KUBERNETES_` simply reads much better than `IFK`.